Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Instance/Instance.h

```diff
@@ -52,6 +52,12 @@ namespace AzToolsFramework
         using EntityOptionalReference = AZStd::optional<AZStd::reference_wrapper<AZ::Entity>>;
         using EntityOptionalConstReference = AZStd::optional<AZStd::reference_wrapper<const AZ::Entity>>;
 
+        enum class EntityIdInstanceRelationship : uint8_t
+        {
+            OneToOne,
```

Comment:
A comment here would be great explaining what is going on.

Reply:
I'll investigate! Looping in @AMZN-koppersr @srikappa-amzn @michabr as this is their domain.